src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private final String[] keys = new String[]{"supply", "buy"};
+    private final int[] values = new int[keys.length];
```

Comment:
follow stateless implementation principles. Don’t take the state at the class level.